src/perf/lib/Tcp.cpp

```diff
@@ -49,44 +49,55 @@ const uint8_t FixedIv[CXPLAT_MAX_IV_LENGTH] = { 0 };
 
 const QUIC_HKDF_LABELS TcpHkdfLabels = { "tcp key", "tcp iv", "tcp hp", "tcp ku" };
 
-struct LoadSecConfigHelper {
-    LoadSecConfigHelper() : SecConfig(nullptr) { CxPlatEventInitialize(&CallbackEvent, TRUE, FALSE); }
-    ~LoadSecConfigHelper() { CxPlatEventUninitialize(CallbackEvent); }
-    CXPLAT_SEC_CONFIG* Load(const QUIC_CREDENTIAL_CONFIG* CredConfig) {
-        if (QUIC_FAILED(
-            CxPlatTlsSecConfigCreate(
-                CredConfig,
-                CXPLAT_TLS_CREDENTIAL_FLAG_NONE,
-                &TcpEngine::TlsCallbacks,
-                this,
-                SecConfigCallback))) {
-            return nullptr;
-        }
-        CxPlatEventWaitForever(CallbackEvent);
-        return SecConfig;
-    }
-private:
-    static
-    _IRQL_requires_max_(PASSIVE_LEVEL)
```

Comment:
Aren't the SAL annotations supposed to stay in sync between the declaration and definition?